winml/api/Windows.AI.MachineLearning.idl

```diff
@@ -213,6 +213,15 @@ namespace ROOT_NS.AI.MachineLearning
         //! BatchSizeOverride > 0 indicates the size of batch that will be used to override the model batch size and optimize evaluations.
         UInt32 BatchSizeOverride { get; set; };
 
+        [contract(MachineLearningContract, 4)]
+        {
+            //! The OverrideNamedDimension method will allow the model compiler to use constant batch size performance optimizations when setting up the LearningModelSession.
+            //! The caller can specify the size of the dimension for a given named dimension.
+            //! dimension = 0 indicates that the dimension present in the model should be honored.
+            //! dimension > 0 indicates the size of the dimension that will be used to override the model "name" dimension and optimize evaluations.
+            void OverrideNamedDimension(String name, UInt32 dimension);
```

Comment:
I used UInt32, not UInt64, since the internal Ort API uses uint32

Reply:
hmm, we set it to int64 as we tend to use int64 in all of the other places in the api

Reply:
Seems like we use int32 for the batch size override, which is probably the most similar API to this one

Reply:
discussed offline, we will match the batchsizeoverride api